VMEncryption/main/DiskUtil.py

```diff
@@ -316,8 +316,10 @@ def mount_all(self):
     def get_mount_items(self):
         items = []
 
-        for line in file('/proc/mounts'):
-            line = [s.decode('string_escape') for s in line.split()]
+        for line in open('/proc/mounts'):
+            #P3TODO - used to be s.decode('string_escape') need to update this to be both python2 and 3 compatible
```

Comment:
check python2/python3

Reply:
adding a python2 + python3 compatible open and unescape routine and addressing todo comment